sort-out-dist.sh

```diff
@@ -24,7 +24,7 @@ do
     #echo "# ${file}"
     base=$(basename $file)
     # rver is '66-1'
-    rver=$(echo $file | grep "^icu-rrelease-[0-9][0-9]-[0-9]" | cut -d- -f3-4)
+    rver=$(echo $file | grep "^icu-rrelease-[0-9][0-9]-[0-9rc]" | cut -d- -f3-4)
```

Comment:
Possibly a little tighter? Just guessing.
```suggestion
    rver=$(echo $file | grep "^icu-rrelease-[0-9][0-9]-[0-9](rc)?" | cut -d- -f3-4)
```

Reply:
No, the suggestion doesn't work with the file names that are created. Leaving as is!